specs/integrations/synthetics.md

```diff
@@ -0,0 +1,119 @@
+## Synthetics Integration
+
+Synthetic monitors play a crucial role in periodically checking the status of your services and applications on a global scale. General documentation about synthetic monitors can be found in
+[Synthetics getting started page](https://www.elastic.co/guide/en/observability/current/synthetics-get-started.html).
+
+This integration goes in to more detail about how the sythetics monitors would
+be correlated with the APM traces. Synthetics traces can be categorized in to two
+main types
+  1. HTTP checks - These have one-one mapping with APM transactions
+  2. Browser checks - These have a one-to-many mapping with APM transactions
+
+### Correlation
+
+The Synthetics agent (including Heartbeat) takes the responsibility of creating the
+[`traceparent`](../agents/tracing-distributed-tracing.md#trace_id-parent_id-and-traceparent)
+header for each outgoing network request associated with a test during every
+monitor execution.
+
+- `trace.id` and `parent.id`
+  - outgoing requests that are being explicity traced by the synthetics agent
+     will have the `parent.id` and `trace.id` as part of the trace context.
+  - must be unique for each step for a browser monitor
+  - must be unique for a http monitor
+- `sampled` Flag
+  - used to control the sampling decision for all the downstream services.
+  - 100% sampling when tracing is enabled
+
+#### Browser checks
+
+When executing a Synthetics journey with APM tracing enabled for specific URLs
+using the --apm_tracing_urls flag, the Synthetics agent takes the following
```

Comment:
Is there an option to enable tracing globally, for example by setting `--apm_tracing_urls "*"`? A nit here is that I would maybe avoid mentioning APM and just go with tracing. I think this is more in line with our future direction where we don't have APM as a top-level navigation item and it's more around showing signals in the context of a non-APM specific "service" concept.

Reply:
Yes `*` would add the tracing for all outgoing requests. let me clarify that and also remove `APM` from the doc.